pennylane/data/utils.py

```diff
@@ -159,32 +159,30 @@ def _write_prog_bar(progress, completed, barsize, barlength, total_length):
     sys.stdout.flush()
 
 
-def _validate_params(data_type, description, attributes=None):
+def _validate_params(data_type, description, attributes):
     r"""Validate parameters for loading the data"""
 
-    if data_type not in list(DATA_STRUCT.keys()):
+    data = DATA_STRUCT.get(data_type)
+    if not data:
         raise ValueError(
-            f"Currently we have data hosted from types: qchem and qspin, but got {data_type}."
+            f"Currently we have data hosted from types: {DATA_STRUCT.keys()}, but got {data_type}."
```

Comment:
DATA_STRUCT will need to be updated when we add a new data type, e.g. "photondata" or something. Locks us into the PennyLane release cycles. @obliviateandsurrender let's take care of this in another PR.

Reply:
contents of DATA_STRUCT should be moved to a config file saved in S3, as should the contents of the `_s3_filemap`. Agreed that we can work out those details after this PR